lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -3207,6 +3336,13 @@ instance IsServerError ErrOutputTokenQuantityExceedsLimit where
         , "."
         ]
 
+instance IsServerError ErrCreateMigrationPlan where
+    toServerError = \case
+        ErrCreateMigrationPlanEmpty ->
+            -- TODO: Provide a more useful error message:
+            apiError err403 NothingToMigrate "Nothing to migrate"
```

Comment:
Provide a more detailed error message here.

At the very least, it needs to explain the types of circumstances that might lead to this error, so the user can take action.

Reply:
Fixed in 713990db02026c286437a7061b275ba41fc72a3e.